sysutils/clamav-server/files/ClamavScanIt.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+#
+# clamdscan or clamscan applied to $*
+#
+
+# License LGPL-3, https://www.gnu.org/licenses/gpl-3.0.en.html
```

Comment:
Is `LGPL` a typo?

Reply:
Yep—copy-and-paste from Portfiles. Fixed. Thanks.